src/keys/slip10.rs

```diff
@@ -122,7 +122,7 @@ pub mod secp256k1 {
                 if scalar.is_zero().into() {
                     false
                 } else {
-                    key_bytes[1..].copy_from_slice(&scalar.to_bytes());
+                    key_bytes[1..].copy_from_slice(&*scalar.to_bytes());
```

Comment:
This doesn't look clippy friendly

Reply:
Weird, my workspace doesn't like it as-is. Oh well.